generators/partials/footer.fsx

```diff
@@ -21,7 +21,7 @@ let footer =
         script [Src "/static/js/modernizr.custom-3.6.0.js"] []
         script [Src "/static/js/learn.js"] []
         script [Src "/static/js/hugo-learn.js"] []
-        link [Rel "stylesheet"; Href "/static/mermaid/mermaid.css"]
-        script [Src "/static/mermaid/mermaid.js"] []
+        // link [Rel "stylesheet"; Href "/static/mermaid/mermaid.css"]
+        script [Src "https://cdnjs.cloudflare.com/ajax/libs/mermaid/9.3.0/mermaid.min.js"] []
```

Comment:
Mermaid was out of date, used this cdn, but I can download it locally if desired.